src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -15,16 +15,60 @@ internal static partial class ErrorFacts
         private const string s_descriptionSuffix = "_Description";
         private static readonly Lazy<ImmutableDictionary<ErrorCode, string>> s_helpLinksMap = new Lazy<ImmutableDictionary<ErrorCode, string>>(CreateHelpLinks);
         private static readonly Lazy<ImmutableDictionary<ErrorCode, string>> s_categoriesMap = new Lazy<ImmutableDictionary<ErrorCode, string>>(CreateCategoriesMap);
-        public static readonly ImmutableHashSet<string> NullableFlowAnalysisWarnings;
+        public static readonly ImmutableHashSet<string> NullableWarnings;
 
         static ErrorFacts()
         {
             ImmutableHashSet<string>.Builder builder = ImmutableHashSet.CreateBuilder<string>();
+
+            builder.Add(getId(ErrorCode.WRN_ConvertingNullableToNonNullable));
```

Comment:
I realized this is full of dupes, will fix (probably by using a minimal diff and adding the new ones to the end)